resources/views/components/user/profile-meta.blade.php

```diff
@@ -19,6 +20,11 @@
 $jrDevPermission = Permissions::JuniorDeveloper;
 
 $isUserStatsDefaultExpanded = request()->cookie('prefers_hidden_user_profile_stats') !== 'true';
+$me = Auth::user() ?? null;
+$amIModerator = false;
+if ($me) {
+    $amIModerator = $me->getAttribute('Permissions') >= Permissions::Moderator;
+}
```

Comment:
can be shortened to something like this:
`$userIsModerator = request()->user()?->getAttribute('Permissions') >= Permissions::Moderator;`

I find the term "me" counter-intuitive in these context. As usually there's no need to reference "me" as most abilities are checked on the acting "user" attached to the request, which in turn exposes `can()` or acting on `request()->user()` 🤔 

Anyways, I think this could be inlined as with the new permissions/roles in place it would probably be rewritten to a blade directive, something like this:

```php
@can('manage', User::class)
    <x-user.profile.moderation-tools :targetUser="$user" />
@endcan
```

Reply:
Here's a good illustration for why I went with the "me" nomenclature:

```php
@if ($userIsModerator)
    <x-user.profile.moderation-tools :targetUser="$user" />
@endif
```

With `$userIsModerator`, it's ambiguous who the `targetUser` prop is targeting. While skimming, I'd infer this to mean we're only referring to a single user, when in fact we're talking about two different users.

The only other way I can think of to make this less ambiguous would be to inline the `request()->...` in the `@if` directive.

Reply:
Ah, I see - that's what i usually referred to as `$actingUser` or similar in such cases.
Makes more sense now that you reminded me 👍

Reply:
I'd probably do `$canModerate` or something similar, which leans into the `@can('manage', User::class)` logic, but I'm fine with this for the time being.